src/config/config_chicago.yml

```diff
@@ -0,0 +1,108 @@
+# City name
+city: Chicago,IL,USA
+# City centerpoint latitude & longitude (default geocoded values set)
+city_latitude: 41.884250000000065
+city_longitude: -87.63244999999995
+
+# City's time zone: defaults to the local time zone of computer initializing the city's config file
+timezone: America/Chicago
+# Radius of city's road network from centerpoint in km, required if OSM has no polygon data (defaults to 20km)
+city_radius: 20
+speed_unit: kph
```

Comment:
You'll probably want to change this to mph, otherwise when you display the visualization, it will show the speed limit slider as kph